### ISSUE TITLE
To set deadlines for publications preparation

### ISSUE THREAD

**Issue by Vovcick** [vbonvin/IPT-public#14]
We need somehow to activate this work

**Comment by vbonvin** [comment]
I'm still a bit in the fog concerning how this project is going. @DSuchet, could you summarize how the things are going ?

**Comment by DSuchet** [comment]
Yep. Sorry for the delay, I've just been awarded a post doc grand and struggling with the administration to get a visa...

**Global overview**
I believe that a journal published by EDP science and intended to a student reader/authorship will be created, with special volumes dedicated to the IPT every year.

**From the editor point of view:** 
I have an appointment with EDP Science, the French Physical Society and Maxime on the 17th. The main objectives are:
1. Decide who is going to pay what
2. Discuss the constitution of the editorial board
3. Find a way to have a team working on the project, since I won't be able to carry out all the work by myself

**From the papers point of view:** 
So far, I counted 22 papers proposals, and received a first draft of 3 of them. 2 were at a satisfying level to enter a reviewing process (considering the aim and scope of the journal). One was significantly not good enough.

Considering the deadline question, I can give you more specific directions after the 17th.
